jekyll/_cci2/high-availability.md

```diff
@@ -6,7 +6,7 @@ order: 20
 description: "Configuring High Availability for CircleCI 2.0"
 ---
 
-**NOTE: Your CircleCI license must include support for high-availability in order to use this configuration. Please contact your account manager if you have questions.**
+**Note: To use this configuration, your CircleCI license must include support for high availability. Please contact your Customer Success Manager if you'd like to set up a High Availability Installation on CircleCI 2.0.**
```

Comment:
I would add a section for this and reword it to make it stronger:

##Prerequisites

Before you configure an existing CircleCI installation for high availability, you must update your license by contacting the Customer Success Manager on your account or by opening a [support ticket]( https://support.circleci.com/hc/en-us/requests/new). Configuring an existing CircleCI installation for HA without first updating the license through the CircleCI customer success team is **not** supported.